Jenkinsfile

```diff
@@ -15,17 +15,6 @@ pipeline {
                 '''
             }
         }
-        stage('Build Docs') {
-            steps {
-                sh '''
-                    sudo make -C doc clean
-                    make -C doc theme
-                    sg docker -c "
-                        docker run --rm -v $(pwd -P)/doc:/doc florianholzapfel/docker-alpine-sphinx make -C /doc html
-                    "
-                '''
-            }
-        }
         stage('Publish Docs') {
```

Comment:
if we are removing build docs step, then we should also remove publish docs step

Reply:
yes, and then the whole pipeline doesn't make sense )

Reply:
Then let's remove whole pipeline :)